### DISCUSSION TITLE
Recursive search

### DISCUSSION THREAD

**Discussion by Potatof** [jcorporation/myMPD#1343] [Ideas]
Add (optional?) recursive search

**Comment by jcorporation** [comment]
I do not understand, please describe your feature request in more detail.

**Reply by Potatof** [reply]
![Sans titre](https://github.com/user-attachments/assets/e0538856-df3e-4268-8363-54ff2c890507)
When searching with the web interface, the result is empty if the music files are in subfolders.
I would like the search to display all files present in subfolders.

**Reply by jcorporation** [reply]
You can use the normal song search to search all files. The filesystem view is not designed for recursive search.
